assets/js/checkout.js

```diff
@@ -28,6 +28,22 @@
             .on('ajaxFail ajaxDone', this.options.formSelector, $.proxy(this.onFailCheckoutForm, this))
     }
 
+    Checkout.prototype.validateCheckout = function ($checkoutForm, callbackFn) {
+        $checkoutForm.request($checkoutForm.data('validateHandler'))
+            .then((response) => {
+                if (!response.error) {
+                    if (!callbackFn)
```

Comment:
I don't see where this callbackFn is passed, what am I missing?

Reply:
Its an optional param, I added it as its needed to allow payment buttons (see other PR) to call the validation as its needed for those.